### ISSUE TITLE
Incorrect Outline with C++20 namespace ::inline syntax

### ISSUE THREAD

**Issue by sean-mcmanus** [microsoft/vscode-cpptools#7216]
The examples I gave all look like they have been fixed, however I found a new example that is not outlined correctly:
```cpp
namespace foo::bar {
    namespace baz::inline boop {
        class beep {
        public:
            beep();
        };

        int foobarbaz();
    }
}
```
![Screenshot_2021-03-23_10-11-39](https://user-images.githubusercontent.com/58710639/112160067-2ffca400-8bc0-11eb-8d19-8d21251f78d5.png)

Of course, this syntax is new to c++20, so it is understandable that it's not fully supported yet. Using the pre-c++20 syntax yields the right outline:
```cpp

namespace foo::bar {
    namespace baz {
        inline namespace boop {
            class beep {
            public:
                beep();
            };

            int foobarbaz();
        }
    }
}
```

_Originally posted by @BigBahss in https://github.com/microsoft/vscode-cpptools/issues/6830#issuecomment-804935850_

**Comment by sean-mcmanus** [comment]
@BigBahss Thanks for reporting this. I've filed a bug against our shared VS parser at https://developercommunity.visualstudio.com/t/C20-tag-parser-doesnt-work-with-names/1379199 . They usually prioritize C++20 features.

**Comment by BigBahss** [comment]
Thanks Sean 👍

**Comment by sean-mcmanus** [comment]
@BigBahss Fixed in 1.13.1: https://github.com/microsoft/vscode-cpptools/releases/tag/v1.13.1
